Package.swift

```diff
@@ -16,7 +17,8 @@ let package = Package(
 	],
 	targets: [
 		.target(
-			name: "Preferences"
+			name: "Preferences",
+            resources: [.process("Resources")]
```

Comment:
The docs do not say this is required: https://developer.apple.com/documentation/xcode/localizing-package-resources

Reply:
Right, Xcode knows how to handle common Apple resource types out-of-the-box. but since the resources folder can evolve, in my point of view is a good idea to declare resources as part of the target definition.

Reply:
I disagree. It can be added later if it actually becomes needed.